lib/puppetx/simp/compliance_mapper.rb

```diff
@@ -73,12 +73,16 @@ def enforcement(key, options = {"mode" => "value"}, &block)
               profile_compiler   = compiler_class.new(self)
 
               profile_compiler.load(&block)
+	      
+	      if options.key?("enabled_sce_versions")
```

Comment:
Reviewing [the failed Travis CI spec test](https://travis-ci.org/simp/pupmod-simp-compliance_markup/jobs/545403016#L1860) (requested by @brandonrdn):

* Some `pry` spelunking shows that the `enabled_sce_versions` setting is absent from the `options` hash at the line during the test failure . 
*  I forced a `backtrace` from here, and found `enforcement()` had been called from https://github.com/simp/pupmod-simp-compliance_markup/blob/9eaf9d35de0057a9836a361bfa7ca800dda86491/lib/hiera/backend/simp_compliance_enforcement_backend.rb#L44-L47
* That method call above only provides the `key`  argument to `enforcement(key, options = {"mode" => "value"}, &block)`.

  Consequently:
  1. `options` defaults to `{"mode" => "value"}` 
     * (and the `enabled_sce_versions` key is left unset)
  2. `enabled_sce_versions` (which is unset) defaults to `[2]` the first time the code calls the `enabled_sce_versions` method: https://github.com/simp/pupmod-simp-compliance_markup/blob/9eaf9d35de0057a9836a361bfa7ca800dda86491/lib/puppetx/simp/compliance_mapper.rb#L527-L530